Class-8-(Intro-to-Spring-and-Controllers).md

```diff
@@ -0,0 +1,34 @@
+# Class 8: Introduction to Spring Boot, MVC, and Controllers
+
+## In the prep work this week, the students learned
+
+1. About the MVC design pattern.
+1. What the Spring framework is.
+1. How to initialize a Spring Boot project.
+1. How to handle routing and write controllers in Spring
+
+## Announcements
+
+1. Check with your course manager for any important announcements.
+
+## Large Group Time (Instructor)
+
+### Class 8 Topics That Require Careful Attention
+
+1. The Java/Gradle development environment can be tricky. Some students may encounter that we were not able to anticiapte in the text. Some minor versions may be different than what is listed in the 
+text. 
+
+   1. Java 13 and Gradle 5.6 are the most crucial to have. 
+   1. If a student has an environment or build issue, do your best to reassure them that these types of configuration hurdles can be common. 
+
+1. Lesson 8 content includes a number of short videos meant to be coding-walkthroughs. Remind the students that it is critical to code along with these. We introduce some IntelliJ tips they may find useful, as well as provide visual aid for the controller tasks we given them.
```

Comment:
Typo fixes:
1) No hyphen in coding walkthroughs.
2) "...visual aids..."
3) "...we give..."